app/views/courses/new.html.erb

```diff
@@ -0,0 +1,20 @@
+<% provide :title, t("course.name_coures") %>
+<h1><%= t("course.create_my_course").upcase %></h1>
+<div class="row">
+<div class="col-md-6 col-md-offset-3">
+  <%= form_for(@course) do |f| %>
+    <%= render "shared/error_messages" %>
+    <%= f.label t("course.name_coures") %>
+    <%= f.text_field :name, class: "form-control" %>
+    <%= f.label t("course.description_course") %>
+    <%= f.text_field :description, class: "form-control" %>
+    <%= f.label t("course.start_date_course") %>
+    <%= f.date_field :start_date, as: :date, value: f.object.try(:strftime, "%m/%d/%Y"), class: "form-control" %>
+    <%= f.label t("course.due_date_course") %>
+    <%= f.date_field :due_date, as: :date, value: f.object.try(:strftime, "%m/%d/%Y"), class: "form-control" %>
+    <%= f.label t("course.status") %>
+    <%= f.select :status, (Course.statuses.keys.map { |w| [w.humanize, w] }), {}, class: "form-control" %>
```

Comment:
code chổ này là sao

Reply:
<%= f.select(:status, [['in process', :in_process], ['Done', :done], ['Admin', :admin]]) %>
thấy nó dễ hiểu hơn nhưng mà nó chạy đúng thì làm đi

Reply:
cái này hắn duyệt qua all params enum của status, rồi sẽ hash cái value status đó ra cái tên status mình muốn select bằng .humanize

Reply:
`Course.statuses.keys.map { |w| [w.humanize, w] }` -> viết vào 1 method trong helper, ở view ko dc connect DB nhé